AirportTicketBookingSystem/AirportTicketBookingSystem/Models/Booking.cs

```diff
@@ -0,0 +1,24 @@
+﻿using AirportTicketBookingSystem.Interfaces;
+
+namespace AirportTicketBookingSystem.Models
+{
+    public class Booking
```

Comment:
do you need 'id's for these models? to distinguish each record and treat it as a unique id.